test/fixtures/e2e/amp-carousel/0.1/multi-visible-single-advance.amp.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1" />
+    <link rel="preload" as="script" href="https://cdn.ampproject.org/v0.js" />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+    <style amp-boilerplate>
```

Comment:
suggestion: If at all possible, shrinking the boilerplate in HTML files can make them more readable. If using VS Code this means saving without formatting (`Cmd + K` + `S`)

Reply:
Any shortcut/method for shrinking boilerplate code?

Reply:
Usually I copy from another `/example` file, but you can also get it from amp.dev:
https://amp.dev/documentation/guides-and-tutorials/learn/spec/amp-boilerplate/